photon-api/src/main/scala/com/linkedin/photon/ml/optimization/game/RandomEffectOptimizationProblem.scala

```diff
@@ -43,27 +43,57 @@ protected[ml] class RandomEffectOptimizationProblem[Objective <: SingleNodeObjec
     val isTrackingState: Boolean)
   extends RDDLike {
 
+  /**
+   * Get the Spark context.
+   *
+   * @return The Spark context
+   */
   def sparkContext: SparkContext = optimizationProblems.sparkContext
```

Comment:
I think `sc` is good for the name of the `SparkContext` member, but I think methods to access `sc` should still be called `sparkContext`.